libtransmission/peer-msgs.cc

```diff
@@ -961,7 +966,7 @@ void sendLtepHandshake(tr_peerMsgsImpl* msgs)
     // http://bittorrent.org/beps/bep_0010.html
     // An integer, the number of outstanding request messages this
     // client supports without dropping any. The default in in
-    // libtorrent is 250.
+    // libtorrent is 2000.
```

Comment:
Sure enough. Looks like this was bumped to 2000 in 834f477e0. Good example of why we shouldn't document other libraries in libtransmission. :smile_cat: Let's just end this source comment at "without dropping any."